src/libraries/Microsoft.Extensions.Hosting.WindowsServices/test/UseWindowsServiceTests.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
This is the wrong license header: https://github.com/dotnet/runtime/blob/0a5ee51f72196253fb5b9c7207451123db631c62/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs#L1-L2

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```